### ISSUE TITLE
[Accessibility] The "Categories Filter" control under the map view is not keyboard-accessible

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4965]
## Describe the bug

* The categories filter in the "Map View" is not keyboard-focusable and operable. Users are unable to choose (ticking the checkboxes) from the the categories

## To Reproduce

1. Launch the local host
2. Navigate to the "Map View"
3. Find the categories filters under the Map View and try to interact with it without using the mouse.
4. Observe if you are able to complete these actions using the keyboard only.

## Expected behavior

Categories filter should be keyboard-focusable and operable, allowing users to interact with it

## Screenshots

![Screenshot from 2024-06-18 18-09-00](https://uploads.linear.app/f1df6e34-3bca-4ade-bb48-e6a457e64091/c3f94f5e-e540-4183-bcdc-75d9c6e590fb/a02cd246-4985-4bf2-af91-490858c075b2?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2YxZGY2ZTM0LTNiY2EtNGFkZS1iYjQ4LWU2YTQ1N2U2NDA5MS9jM2Y5NGY1ZS1lNTQwLTQxODMtYmNkYy03NWQ5YzZlNTkwZmIvYTAyY2QyNDYtNDk4NS00YmYyLWFmOTEtNDkwODU4YzA3NWIyIiwiaWF0IjoxNzE4OTYzMzY4LCJleHAiOjMzMjg5NTIzMzY4fQ.TnTcVE1mUlvjYvSlwEC_KVhyEPDG0ylJsbwmcdTmO0A)

**Comment by Angamanga** [comment]
Closing but keeping in Linear: https://linear.app/ushahidi/issue/USH-1437/[accessibility]-the-categories-filter-control-under-the-map-view-is
